products/views.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models      import Image,SubCategoryProduct,Discount,Product
+    
+class ProductlistView(View):
+    def get(self,request,sub_category_id):
+        try:
+            sub_category_products = SubCategoryProduct.objects.filter(sub_category_id=sub_category_id)
```

Comment:
```suggestion
            products = Product.objects.filter(sub_category__id=sub_category_id)
```
이렇게 수정해보시고, 동작 확인 해보세요!

Reply:
수정했습니다!!